core/connector/src/main/java/org/exoplatform/ecm/connector/dlp/FileDlpConnector.java

```diff
@@ -104,6 +104,23 @@ public boolean processItem(String entityId) {
     return true;
   }
   
+  private boolean itemExist(String entityId) {
+    ExtendedSession session;
+    boolean result = false;
+    try {
+      session = (ExtendedSession) WCMCoreUtils.getSystemSessionProvider().getSession(COLLABORATION_WS, repositoryService.getCurrentRepository());
```

Comment:
Please close the session at the end of this method

Reply:
As we use the sessionProvider which use a cache for session, we dont need to close the session.
The session will be used (and will be the same as long as it is valid). When this session will timeout, it will be closed and removed from the cache.
The sessionprovider manages this session.

Reply:
Yes I know . But we have encountered the same problem before 
https://github.com/exoplatform/ecms/commit/f58ac94ae7837dec6c20299d939b654338efa7e0

Reply:
ok, updated